Tests/TuistDependenciesAcceptanceTests/DependenciesAcceptanceTests.swift

```diff
@@ -109,3 +125,55 @@ final class DependenciesAcceptanceTestAppWithAirshipSDK: TuistAcceptanceTestCase
         try await run(BuildCommand.self)
     }
 }
+
+final class DependenciesAcceptanceTestPackageWithRegistryAndAlamofire: ServerAcceptanceTestCase {
```

Comment:
Since the tests in this file interact with a global state, which can become a source of flakiness, what do you think if we write them using Swift Testing and use [this](https://developer.apple.com/documentation/testing/parallelization) to mark them to run serially?

Reply:
Any test in this test suite can't be run in parallel as they all end up resolving packages – and you can't resolve dependencies at the same time. That's why they are all a single test suite which we don't parallelize.

I agree we should eventually rewrite these with Swift Testing, but given the setup here is not trivial, I'd rather do it in a separate PR dedicated to that.